build.sh

```diff
@@ -122,6 +122,13 @@ then
     build_args+=" /p:BootstrapBuildPath=${bootstrap_path}"
 fi
 
+# https://github.com/dotnet/roslyn/issues/23736
+UNAME="$(uname)"
+if [[ "$UNAME" == "Darwin" ]]
+then
+    build_args+=" /p:UseRoslynAnalyzers=false"
```

Comment:
Can you explain what's up with the analyzers?

Reply:
No idea. That's why I filed the bug to track this down 😄